src/twisted/python/filepath.py

```diff
@@ -1753,17 +1755,15 @@ def statinfo(self, value=_SpecialNoValue):
         # definition - but the getter and setter both need the same function
         # in order for the 'deprecated' decorator to produce the right
         # deprecation string.
-        if value is _SpecialNoValue:
-            return self._statinfo
-        else:
-            self._statinfo = value
+        return self._statinfo
 
 
-    @statinfo.setter
+    # Ignore mypy error due to: https://github.com/python/mypy/issues/1362
+    @statinfo.setter  # type: ignore[misc]
     @deprecated(Version('Twisted', 15, 0, 0),
                 "other FilePath methods such as getsize(), "
                 "isdir(), getModificationTime(), etc.")
-    def statinfo(self, value=_SpecialNoValue):
+    def statinfo(self, value):
```

Comment:
this can also be removed?

Reply:
I've removed this from here, and moved it to: https://github.com/twisted/twisted/pull/1326